iceberg_rust_ffi/Cargo.toml

```
@@ -12,7 +12,7 @@ default = ["julia"]
 julia = []
 
 [dependencies]
-iceberg = { git = "https://github.com/RelationalAI/iceberg-rust.git", rev = "37e79b805407a0340d08823373cafed9cdac0083" }
+iceberg = { git = "https://github.com/RelationalAI/iceberg-rust.git", rev = "21244af20e1c072deb6d8daf56a328dd8e12a545" }
```

Can you make this the latest commit? Currently ae83309fd198ec30f052a7e9f983711c5f581aea
```suggestion
iceberg = { git = "https://github.com/RelationalAI/iceberg-rust.git", rev = "ae83309fd198ec30f052a7e9f983711c5f581aea" }
```